aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -187,13 +187,22 @@ pub enum EntryFunctionCall {
     },
 
     /// Creates a new resource account, publishes the package under this account transaction under
-    /// this account and leaves the signer cap readily available for pickup.
+    /// this account, and leaves the signer cap readily available for pickup.
```

Comment:
This comment needs to more clearly explain what this struct is for... What do you mean "[The struct] Creates a new resource account"?